Makefile

```diff
@@ -6,9 +6,6 @@ clean:
 	rm -rf ~/.dagster_home; mkdir ~/.dagster_home; cp dagster.yaml ~/.dagster_home/dagster.yaml
 
 manifest:
```

Comment:
This command is run in the context of the GHA runner in order to build the manifest, so these installs are necessary.

Reply:
yeah I realized that too late -- that said, I'd prefer to put the commands in the GHA vs. in the make file (since you might run `make manifest` in the context of local dev, in which case we don't need the pip installs (handled via the `setup.py`)